promotion/src/main/resources/db/changelog/ddl/changelog-0002.sql

```diff
@@ -0,0 +1,4 @@
+--liquibase formatted sql
+
+--changeset vonhu:issue-promotion-0002
+alter table if exists promotion add column is_active boolean;
```

Comment:
add blank line

Reply:
why I should add blank line in this file?

Reply:
The end-of-line issue is detected by Github.
Refer https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log
All of the files should have a newline.